tools/cldr-apps/js/src/views/InfoPanel.vue

```diff
@@ -1,25 +1,54 @@
 <template>
-  <section id="InfoPanelSection">
-    <header class="sidebyside-column-top">
-      <button
-        class="cldr-nav-btn info-panel-closebox"
-        title="Close"
-        @click="closeInfoPanel"
-      >
-        ✕
-      </button>
-      <span class="i-am-info-panel">Info Panel</span>
-      <button class="cldr-nav-btn" title="Reload" @click="reloadInfoPanel">
-        ↻
-      </button>
-    </header>
-  </section>
+  <div>
+    <section id="InfoPanelSection">
+      <header class="sidebyside-column-top">
+        <a-button
+          shape="circle"
+          class="cldr-nav-btn info-panel-closebox"
+          title="Close"
+          @click="closeInfoPanel"
+        >
+          ✕
+        </a-button>
+        <span class="i-am-info-panel">Info Panel</span>
+        <a-button
+          shape="circle"
+          class="cldr-nav-btn"
+          title="Reload"
+          @click="reloadInfoPanel"
+        >
+          ↻
+        </a-button>
+        <a-button
+          shape="circle"
+          class="cldr-nav-btn"
+          title="Reload"
```

Comment:
"Reload" is wrong here -- should be something like "Explain inheritance"

Reply:
When this is extended to include "bailey explainer", there should be an explanation of the bailey value, as well as an explanation of the winning value (if they're not the same) -- I guess a single control (button) to show both is better than separate controls attached to the winning and bailey values

Reply:
> When this is extended to include "bailey explainer", there should be an explanation of the bailey value, as well as an explanation of the winning value (if they're not the same) -- I guess a single control (button) to show both is better than separate controls attached to the winning and bailey values

I'd like some more clarity on what that looks like. We don't surface the Bailey value explicitly as such however it's always one of the candidates. Maybe clicking on that candidate and then clicking this gets you there.

Reply:
"We don't surface the Bailey value explicitly as such however it's always one of the candidates" -- I'd say that does count as surfacing -- we just don't surface the term "Bailey"

I still don't really get the point of "inheritance explainer" for a value that's not inherited -- there's not much to explain other than the obvious fact that since it's not inherited it must be present in the current locale+path